docs/modules/publisher.md

```diff
@@ -319,26 +345,27 @@ Returns the nodes configured for one Endpoint (Dataset)
 
   *Example:*
   >*Method Name:* `GetConfiguredNodesOnEndpoints_V1`\
-  *Payload:* 
+  >  *Payload:* 
   >``` json
   >{
-  >    "EndpointUrl": "opc.tcp://192.168.100.20:50000"
+  >"EndpointUrl": "opc.tcp://192.168.100.20:50000"
   >}
   >```
   >*Response:*  
+  >
   >``` json
   >{
-  >    "status":200,
-  >    "payload":
-  >    [
-  >       {
-  >         "id":"nsu=http://microsoft.com/Opc/OpcPlc/;s=SlowUInt1",
-  >         "opcSamplingInterval":3000,
-  >         "opcSamplingIntervalTimespan":"00:00:03",
-  >         "heartbeatInterval":0,
-  >         "heartbeatIntervalTimespan":"00:00:00"
-  >       }
-  >    ]
+  >"status":200,
+  >"payload":
+  >[
+  >  {
+  >    "id":"nsu=http://microsoft.com/Opc/OpcPlc/;s=SlowUInt1",
+  >    "opcSamplingInterval":3000,
+  >    "opcSamplingIntervalTimespan":"00:00:03",
+  >    "heartbeatInterval":0,
+  >    "heartbeatIntervalTimespan":"00:00:00"
+  >  }
+  >]
   >}
   >```
```

Comment:
any particular reason for these changes?

Reply:
I fixed formatting of all direct methods which was inconsistent.